Facebook.AudienceNetwork.iOS/source/Facebook.AudienceNetwork/ApiDefinition.cs

```diff
@@ -830,4 +858,89 @@ interface InstreamAdViewDelegate
 		[Export ("adViewWillLogImpression:")]
 		void AdViewWillLogImpression (InstreamAdView adView);
 	}
+
+	// @interface FBRewardedVideoAd : NSObject
+	[BaseType (typeof (NSObject), Name = "FBRewardedVideoAd")]
+	interface RewardedVideoAd
```

Comment:
I think this class needs [DisableDefaultCtor] due constructors with placementID, there's no other way to set this property.